src/payloads/AppUser.ts

```diff
@@ -46,3 +47,23 @@ export class AppUserRolesResponse {
   @IsEnum(AppUserRole)
   role: string;
 }
+
+export class AppUserProfileResponse {
+  @IsString()
+  firstName: string;
+
+  @IsString()
+  lastName: string;
+
+  @IsString()
+  email: string;
+
+  @IsString()
+  major: string;
+
+  @IsString()
+  graduationYear: string;
+
+  @IsInstance(InductionClass)
+  inductionClass: InductionClass;
```

Comment:
Should make this eager in InductionClass's entity file to fix the problem you had (https://orkhan.gitbook.io/typeorm/docs/eager-and-lazy-relations) 

An example in our codebase (hosts): https://github.com/HKN-UCSD/hkn-server/blob/dev/src/entities/Event.ts

Reply:
Cool.. We get induction class in our response now.